packages/kbn-esql-utils/src/utils/query_parsing_helpers.ts

```diff
@@ -114,6 +114,19 @@ export const getTimeFieldFromESQLQuery = (esql: string) => {
   return column?.name;
 };
 
+export const isQueryWrappedByPipes = (query: string): boolean => {
+  const { ast } = getAstAndSyntaxErrors(query);
+  const trimmedCode = query.trim().replaceAll('\n  |', '\n|');
+  const numberOfCommands = ast.length;
+  const pipesWithNewLine = trimmedCode?.split('\n|');
```

Comment:
Not sure, it looks the `replaceAll()` might not be necessary:

```suggestion
  const trimmedCode = query.trim();
  const numberOfCommands = ast.length;
  const pipesWithNewLine = trimmedCode?.split('\n  |');
```

Maybe `trim()` as well:

```suggestion
  const numberOfCommands = ast.length;
  const pipesWithNewLine = query.split('\n  |');
```

Reply:
True, adjusted here https://github.com/elastic/kibana/pull/191269/commits/a881caa03ada74975fa7ca3b84caab4a5732d5a6